cloud/docker/docker.py

```diff
@@ -682,6 +682,10 @@ def get_deployed_containers(self):
             name = '/' + name
         deployed = []
 
+        for img in self.client.images():
+            if image in img['RepoTags']:
+                all_valid_tags = set(x.rsplit(':', 1)[1] for x in img['RepoTags'])
+
```

Comment:
I noticed that previously, this could potentially save tags from multiple img's.  Was that correct behaviour?  I kinda think it might have been incorrect.  If so, we should break out of the loop once we've set all_valid_tags.